### PR TITLE
Release 0.15.0

### PR DESCRIPTION
**Pull request by straight-shoota**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
 # CHANGELOG
 
+## v0.15.0 - 2021-06-29
+
+Fixes:
```
Comment (beta-ziliani) [inline CHANGELOG.md:5]:
```suggestion
### Fixes
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+
+- Let `shards build` error if no targets defined ([#490](https://github.com/crystal-lang/shards/pull/490), thanks @straight-shoota)
+- Fix to allow empty `shard.override.yml` ([#495](https://github.com/crystal-lang/shards/pull/495), thanks @straight-shoota)
+- Stop expecting master to be the default branch for git ([#503](https://github.com/crystal-lang/shards/pull/503), thanks @szabgab)
+
+Features:
```
Comment (beta-ziliani) [inline CHANGELOG.md:11]:
```suggestion
### Features

```

**Review by bcardiff — APPROVED** [review]

**Review by beta-ziliani — APPROVED** [review]
I suggested minor edits, but it can still be merged as is 👌 
Thanks @straight-shoota !

File: CHANGELOG.md (hunk 1 above)
Comment (straight-shoota) [reply CHANGELOG.md:5]: I'm following the previous markup in the changelog.

We can change it, but I'd make that a separate PR and update the entire file.

Comment (beta-ziliani) [reply CHANGELOG.md:5]: Alright then

**Pull request merged**